File: week1-c/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get height value from user
    bool error = true;
    int height = 0;
    //ask until valid height value
    while (error)
    {
        height = get_int("Height: ");
        if (height >= 1 && height <= 8)
        {
            error = false;
        }
    }
    //loop for pyramid height
    for (int i = 1; i <= height; i++)
    {
        //loop to draw the pyramid blocks
        for (int base = height-1; base >= 0 ; base--)
        {
            if(base < i){
                printf("#");
            }
            else{
                printf(" ");
            }
        }
        //insert a new line
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
#!/bin/python3
import cs50

#ask until valid height value
while True:
    height = cs50.get_int("Height: ")
    if height >= 1 and height <= 8:
        break
# loop for pyramid height
for i in range (1, height+1):
    # loop to draw the pyramid blocks
    for base in range (height, 0, -1):
        # print(f"i: {i}, b: {base}")
        if base <= i:
            print("#", end="")
        else:
            print(" ", end="")
    # insert a new line
    print("", end="\n")